test/FunctionalTestUtils/Tests/RequestTelemetryTestsBase.cs

```diff
@@ -31,5 +33,25 @@ public void ValidateBasicRequest(string requestPath, RequestTelemetry expected)
             Assert.True(actual.Duration < (DateTimeOffset.Now - testStart), "duration");
             Assert.Equal(expected.HttpMethod, actual.HttpMethod);
         }
+
+        public void ValidateBasicException(string requestPath, ExceptionTelemetry expected)
```

Comment:
Is this test in the right class? It verifies `ExceptionTelemetry` instances while the class seems to imply its for `RequestTelemetry` instances. Alternatively, is this base class needed? Can these two methods be pulled up or pushed down?